test/test_helper.rb

```diff
@@ -1,5 +1,15 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+require 'simplecov-lcov'
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
+  SimpleCov::Formatter::LcovFormatter,
```

Comment:
Layout/IndentFirstArrayElement: Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.

Reply:
This bot was hooked up to my GitHub. Will fix.

Reply:
Fixed.